src/ErrorConsoleTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-tooling/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Tooling;
+
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+trait ErrorConsoleTrait
+{
+    /**
+     * Retrieve console output, if possible, to allow writing to STDERR.
+     *
+     * @param OutputInterface $output
+     * @return OutputInterface
+     */
+    private function getErrorConsole(OutputInterface $output)
```

Comment:
All this fuss just for some errors? Nah, just let the exceptions bubble up, and consumers can use `-vvv` to see all the details